gradle-recommended-product-dependencies/src/main/java/com/palantir/gradle/dist/CompileRecommendedProductDependencies.java

```diff
@@ -17,36 +17,46 @@
 package com.palantir.gradle.dist;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import java.io.File;
 import java.io.IOException;
+import javax.inject.Inject;
 import org.gradle.api.DefaultTask;
+import org.gradle.api.Project;
 import org.gradle.api.file.DirectoryProperty;
+import org.gradle.api.file.RegularFileProperty;
 import org.gradle.api.provider.SetProperty;
 import org.gradle.api.tasks.Input;
 import org.gradle.api.tasks.OutputDirectory;
+import org.gradle.api.tasks.OutputFile;
 import org.gradle.api.tasks.TaskAction;
 
 public abstract class CompileRecommendedProductDependencies extends DefaultTask {
 
+
+    public static final String RESOURCE_PATH =
+            RecommendedProductDependencies.SLS_RECOMMENDED_PRODUCT_DEPS_KEY + "/product-dependencies.json";
     static final ObjectMapper MAPPER = new ObjectMapper();
 
     @Input
     abstract SetProperty<ProductDependency> getRecommendedProductDependencies();
 
+    @OutputFile
+    abstract RegularFileProperty getOutputFile();
```

Comment:
I don't even think we need to make this an `@OutputFile` - could just make it a helper method on the class to get the output.

Reply:
Actually, there is one possible benefit: the dev-env code that calls `outputs.singleFile` would fail harder with the `@OutputFile` setup I think that just a helper method.